src/features/Auth/pages/RestorePass/RestorePass.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useFormik } from 'formik';
+import { useNavigate } from 'react-router-dom';
+import validationSchema from './validationSchema';
+import Input from '../../components/Input/Input';
+import Header from '../../components/Header/Header';
+import ButtonSubmit from '../../components/ButtonSubmit/ButtonSubmit';
+import { StyledForm } from '../../modules/StyledForm';
+import AuthText from '../../components/AuthText/AuthText';
+import { StyledHeaderIcon } from '../../modules/StyledHeaderIcon';
+
+const RestorePass = function () {
+  const navigate = useNavigate();
+
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+    },
+    validationSchema,
+    onSubmit: (values) => {
+      localStorage.setItem(values.email, JSON.stringify(values));
+      navigate('/signin');
+    },
+  });
+
+  return (
+    <StyledForm onSubmit={formik.handleSubmit}>
+      <Header text="Restore Password" isSignUp={false}>
```

Comment:
можно явно не прописывать что у тебя isSignUp=false, если ты его не будешь передавать, он будет undefined, что будет иметь тот же смысл. резонно так передавать если у тебя стоит в компоненте на этом пропсе значение по умолчанию true

```suggestion
      <Header text="Restore Password">
```